### DISCUSSION TITLE
"Viewed" checkbox on bottom of diff

### DISCUSSION THREAD

**Discussion by lucas-nasajon** [community/community#9676] [Pull Requests]
When we have long diffs in a PR, after reviewing them, if we want to mark the diff as viewed, we have to go all the way back to its top to click on the checkbox. I think it could be really useful if the checkbox were moved, or duplicated, to the bottom of the diff.

**Comment by patmacpherson-sc** [comment]
Similar discussion - https://github.com/github/feedback/discussions/4759

+1 from me. It would be a nicer flow - read through the code, and click a checkbox at the bottom of the file.

**Reply by zkuhn** [reply]
similar discussion link gives 404. but +1 on dupe checkbox at bottom request

**Comment by mruhlin** [comment]
From what I can tell, this is a Firefox-specific issue (either that or they're A/B testing something).  I have the same PR open in chrome and Firefox.  In chrome, the "viewed" checkbox hovers at the top of the window, so is always easily clickable.  In Firefox it doesn't and I have to scroll up.

**Reply by zkuhn** [reply]
can't  confirm.. chrome/mac file headers scroll up off page (Jan 2023)

**Reply by rburgst** [reply]
as of a few days ago I also get this on chrome that the "viewed" checkbox is no longer sticky!

**Reply by brother** [reply]
On firefox 132 (linux) here and it started scrolling out of view at some point last few days (haven't been at this computer for a week).
